doc/classes/Vector2.xml

```diff
@@ -259,10 +259,10 @@
 	</methods>
 	<members>
 		<member name="x" type="float" setter="" getter="">
-			The vector's x component.
+			The vector's x component. Also accesible by using the index position `[0]`.
```

Comment:
You can't use Markdown here, it's a pseudo BBCode. So you should use `[code][0][/code]`. Also typo in "acces**s**ible".

Reply:
fixed the Markdown and the typo